amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/AmqpConnection.java

```diff
@@ -102,12 +102,10 @@
     private AtomicBoolean blocked = new AtomicBoolean();
     private AmqpOutputConverter amqpOutputConverter;
     private ServerCnx pulsarServerCnx;
-
-    @Getter
-    private final AmqpTopicManager amqpTopicManager;
+    private AmqpBrokerService amqpBrokerService;
 
     public AmqpConnection(PulsarService pulsarService, AmqpServiceConfiguration amqpConfig,
-                          AmqpTopicManager amqpTopicManager) {
+                          AmqpBrokerService amqpBrokerService) {
```

Comment:
Since the `AmqpBrokerService` already has an instance of the `PulsarService`, can it be removed from the function definition, and be pulled from the `amqpBrokerService.getPulsarService()`?

Reply:
Yes!